#This project uses Illumina RNAseq data to look for differential gene expression in sheep with foot rot

The RNASeq data was taken from a sheep foot biopsy, there are 12 files in total. There are three
samples from a healthy sheep (1, 1b and 1c) and three samples from a sheep affected by footrot
(2a, 2b and 2c). The RNA was extraction following a Trizol extraction methodology
(http://tools.thermofisher.com/content/sfs/manuals/trizol_reagent.pdf) and the sequencing
library was constructed using a whole RNA TruSeq protocol and sequenced as 150bp paired end.
 
https://github.com/addyblanch/bioinformatics_apprenticeship

Module One - Bioinformatics Methods
1. RNASeq Submission file
2. RNASeq Genome file
3. RNASeq Dataset

Data
https://figshare.com/s/109c4c53b42a65e6e74d

Methodology

https://combine-lab.github.io/salmon/getting_started/

Patro, R., Duggal, G., Love, M. I., Irizarry, R. A., & Kingsford, C. (2017). Salmon provides fast and bias-aware quantification of transcript expression. Nature Methods

For calculating differential gene expression online

http://bioinformatics.sdstate.edu/idep/

Ge, S.X., Son, E.W. & Yao, R. iDEP: an integrated web application for differential expression and pathway analysis of RNA-Seq data. BMC Bioinformatics 19, 534 (2018). https://doi.org/10.1186/s12859-018-2486-6
